subprojects/platform-play/src/main/java/org/gradle/play/internal/run/PlayWorkerServer.java

```diff
@@ -85,7 +85,7 @@ private InetSocketAddress startServer() {
         ClassLoaderUtils.disableUrlConnectionCaching();
         final Thread thread = Thread.currentThread();
         final ClassLoader previousContextClassLoader = thread.getContextClassLoader();
-        final ClassLoader classLoader = new URLClassLoader(DefaultClassPath.of(runSpec.getClasspath()).getAsURLArray(), null);
+        final ClassLoader classLoader = new URLClassLoader(DefaultClassPath.of(runSpec.getClasspath()).getAsURLArray(), ClassLoader.getSystemClassLoader());
```

Comment:
On Java 9, bootstrap classloader doesn't contain `java.sql` module, so we have to use system classloader as its parent.